kf_lib_data_ingest/common/warehousing.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+import sqlalchemy
+
+
+def init_study_db(db_host, study_id):
+    """
+    Create and/or reset a warehouse database for the given study.
+
+    :param db_host: Host URL of the warehouse database
+    :type db_host: str
+    :param study_id: Unique identifier for the study
+    :type study_id: str
+    """
+    # create the study database if needed
+    db_url = os.path.join(db_host, "postgres")
```

Comment:
> if someone provides a url like: postgresql://foo:bar@localhost:5431/postgres

They shouldn't do that because the code needs to designate the DB for itself

Reply:
But someone might accidentally do it so we shouldn't we just strip it out and replace it with postgres since we'll always use that db to connect?

Reply:
I think I'd rather go in the other direction and make it not have to necessarily be postgres